src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -417,6 +418,107 @@ class C
         }
 
         [Fact]
+        [WorkItem(217740, "https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems?id=217740")]
+        public void DateTimeConstantAttribute()
+        {
+            #region "Source"
+            var source = @"
+using System;
+using System.Runtime.CompilerServices;
+
+public class Bar
+{
+    public void Method([DateTimeConstant(-1)]DateTime p1) { }
```

Comment:
Yes, it looks like C# never synthesizes the attribute.

---
In reply to: [64610028](https://github.com/dotnet/roslyn/pull/11536#discussion_r64610028) [](ancestors = 64610028)